documents/Libraries/stdlib/sx-osl/color2.h

```diff
@@ -1,244 +0,0 @@
-// Open Shading Language : Copyright (c) 2009-2017 Sony Pictures Imageworks Inc., et al.
-// https://github.com/imageworks/OpenShadingLanguage/blob/master/LICENSE
```

Comment:
These headers are already in the stdlib/osl/ directory so these unused copies can be removed.